### PR TITLE
WPCS 2.0.0: Remove the deprecated VIP ruleset and sniffs

### PR DESCRIPTION
**Pull request by jrfnl**
The VIP ruleset and sniffs were deprecated in WPCS 1.0.0 and will now be removed for WPCS 2.0.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,9 +137,8 @@ You can use the following as standard names when invoking `phpcs` to select snif
...
     - includes `WordPress-Core`
 
-**Notes:** This WPCS package contains the sniffs for another ruleset, `WordPress-VIP`. This ruleset was originally intended to aid with the [WordPress.com VIP coding requirements](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/), but this is no longer used or recommended by the WordPress.com VIP team or their clients, since they prefer to use their [official VIP coding standards](https://github.com/Automattic/VIP-Coding-Standards) ruleset instead.
-
-Before WPCS `1.0.0`, the WordPress-VIP ruleset was included as part of the complete `WordPress` ruleset. **As of `1.0.0` the `WordPress-VIP` ruleset is not part of the WordPress ruleset, and it is deprecated**. The remaining `WordPress-VIP` sniffs may still be referenced in custom rulesets, so to maintain some backwards compatibility, they will remain in WPCS until `2.0.0`. See [#1309](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1309) for more information.
+**Note:** The WPCS package used to a `WordPress-VIP` ruleset and associated sniffs, prior to WPCS 2.0.0.
```
Comment (ntwb) [inline README.md]:
```suggestion
**Note:** The WPCS package used to include a `WordPress-VIP` ruleset and associated sniffs, prior to WPCS 2.0.0.
```

Comment (jrfnl) [reply README.md]: Good catch, thanks! I've fixed up the commit.

File: WordPress/Tests/Arrays/ArrayDeclarationSpacingUnitTest.1.inc [hunk 2]
```diff
@@ -111,6 +111,6 @@ $bad = array( 'key1' => 'value1', 'key2' => 'value2' ); // Bad.
 // @codingStandardsChangeSetting WordPress.Arrays.ArrayDeclarationSpacing allow_single_item_single_line_associative_arrays true
 
 $foo = array(
-	'meta_key'   => 'foo', // phpcs:ignore WordPress.VIP.SlowDBQuery.slow_db_query_meta_key
-	'meta_value' => 'bar', // phpcs:ignore WordPress.VIP.SlowDBQuery.slow_db_query_meta_value
+	'meta_key'   => 'foo', // phpcs:ignore Standard.Category.SniffName.ErrorCode
```
Comment (GaryJones) [inline WordPress/Tests/Arrays/ArrayDeclarationSpacingUnitTest.1.inc]: What's the deal with these placeholders?

Comment (jrfnl) [reply WordPress/Tests/Arrays/ArrayDeclarationSpacingUnitTest.1.inc]: These are test just to see whether the sniff handles the PHPCS ignore annotations correctly, they should never have referenced real sniffs.
As they did and the sniff they refer to is now being removed, it seemed right to clean up the test.

**Pull request merged**